octobot_trading/personal_data/orders/order.py

```diff
@@ -206,13 +206,19 @@ def is_open(self):
         return self.state is None or self.state.is_open()
 
     def is_filled(self):
-        return self.state.is_filled() or (self.state.is_closed() and self.status is enums.OrderStatus.FILLED)
+        if self.state is None:
```

Comment:
How can it be possible ?

Reply:
I has this case when creating tests, you think it can't happen in a real situation ?

Reply:
I doesn't happened until now, and we really don't want that to happen.

Reply:
ok, i'll remove this check then